spec/controllers/email_controller_spec.rb

```diff
@@ -15,7 +15,7 @@
 
     context "with valid accepted applications" do
       before :each do
-        @application = FactoryGirl.create(:application_letter_accepted, event: @event, user: FactoryGirl.build(:user))
+        @application = FactoryGirl.create(:application_letter, :accepted, event: @event, user: FactoryGirl.build(:user))
```

Comment:
Is the user association in this create call even necessary? I think [this association definition](https://github.com/hpi-schuelerklub/workshop-portal/blob/35d5b8dd50ac72141dd33b35200a3682b5712588/spec/factories/application_letters.rb#L22) should be enough?